apps/system/test/unit/app_text_selection_dialog_test.js

```diff
@@ -234,13 +231,19 @@ suite('system/AppTextSelectionDialog', function() {
     assert.isTrue(stubChangeTransitionState.calledWith('opened'));
   });
 
-  test('_resetCutOrCopiedTimer', function() {
+  test('_launchCutOrCopiedTimer', function() {
```

Comment:
These method-level unit tests can be moved accordingly.

We would need another test to ensure two instances are getting the same globalStates instance.